src/poetry/repositories/installed_repository.py

```diff
@@ -239,7 +242,7 @@ def load(cls, env: Env, with_dependencies: bool = False) -> InstalledRepository:
         seen = set()
         skipped = set()
 
-        for entry in reversed(env.sys_path):
+        for entry in env.sys_path:
```

Comment:
Any special reason for that removal?

Reply:
I think it is wrong. Actually, I was wondering why it should be reversed order. The correct order is required to overwrite the version of a global plugin. The following test (I added) will fail with reversed order: `test_loads_in_correct_sys_path_order`